### PR TITLE
proxy_set_header required for admin

### PR DESCRIPTION
**Pull request by dmenne**
See #848 and #1313

## What type of PR?

(Feature, enhancement, bug-fix, documentation)

### Related issue(s)
- Mention an issue like: #001
- Auto close an issue like: closes #001

## Prerequistes
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [ ] In case of feature or enhancement: documentation updated accordingly
- [ ] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/guide.html#changelog) entry file.

### CODE REVIEW

File: docs/reverse.rst [hunk 1]
```diff
@@ -92,7 +93,7 @@ Here is an example configuration :
     # [...] here goes your standard configuration
 
     location /webmail {
-      proxy_pass https://localhost:8443/webmail;
+      proxy_pass https://localhost:8443/webmail;
```
Comment (Nebukadneza) [inline docs/reverse.rst]: nitpick: bogus change due to adding trailing whitespace.

**Comment by delijati** [comment]
can this be triggered again

**Comment by delijati** [comment]
It would be also nice to have working example with `letsencrypt`:
aka: 

```
$ cat /etc/nginx/sites-enabled/mailu.conf 
server {

  server_name mail.foo.de foo.de; # managed by Certbot

  location /admin {
    proxy_pass https://localhost:8443/admin;
    proxy_set_header Host $http_host;
  }

  location /webmail {
     proxy_pass https://localhost:8443/webmail;
  }

  listen [::]:443 ssl; # managed by Certbot
  listen 443 ssl; # managed by Certbot

  ssl_protocols TLSv1.1 TLSv1.2;
  ssl_ciphers 'ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384';
  ssl_prefer_server_ciphers on;
  ssl_session_timeout 10m;
  ssl_session_tickets off;

  ssl_certificate /mailu/certs/letsencrypt/live/mailu/fullchain.pem; # managed by Certbot
  ssl_certificate_key /mailu/certs/letsencrypt/live/mailu/privkey.pem; # managed by Certbot

  # include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
  # ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
}
```

**Comment by ofthesun9** [comment]
> It would be also nice to have working example with `letsencrypt`:
> aka:

I think the first example in the doc is generic enough and covers also your example ? If you think that another exemple need to ne included, feel free tp submit a PR :-)

We might also include a list of reverse proxy known to work ( I am using [https-portal ](https://github.com/SteveLTN/https-portal)), 
We could also mention that two variables can be used to specify the names of the cert file and key, as described in the section [Finish setting up TLS](https://mailu.io/master/compose/setup.html#finish-setting-up-tls)

bors retry

**Comment by ofthesun9** [comment]
bors try

**Comment by ofthesun9** [comment]
Replaced by #1423

**Pull request closed without merging**